visualization/src/pcl_visualizer.cpp

```diff
@@ -2983,7 +2983,7 @@ pcl::visualization::PCLVisualizer::updateColorHandlerIndex (const std::string &i
   int color_handler_size = int (am_it->second.color_handlers.size ());
   if (index >= color_handler_size)
   {
-    pcl::console::print_warn (stderr, "[updateColorHandlerIndex] Invalid index <%d> given! Maximum range is: 0-%lu.\n", index, static_cast<unsigned long> (am_it->second.color_handlers.size ()));
+    pcl::console::print_warn (stderr, "[updateColorHandlerIndex] Invalid index <%d> given! Maximum range is: [0-%d).\n", index, color_handler_size);
```

Comment:
Shouldn't it be `color_handler_size - 1`?

Reply:
Yes, it was wrong there. I have tried to fix that with right-open interval convention `[0-5)`, which makes it now correct, but it is not maybe that natural. I will rework that.